src/sentry/auth/authenticators/u2f.py

```diff
@@ -203,15 +205,28 @@ def activate(self, request: HttpRequest) -> ActivationChallengeResult:
         challenge, state = self.webauthn_authentication_server.authenticate_begin(
             credentials=credentials
         )
-        request.session["webauthn_authentication_state"] = state
+        if request.session.get("staff_u2f", False):
```

Comment:
@schew2381 Where are you setting `staff_u2f` in the session?

Reply:
whoops I linked to the same PR in the description, it's here: https://github.com/getsentry/getsentry/pull/13196